en/deploy-multiple-tidb-operator.md

```diff
@@ -102,13 +102,18 @@ To support deploying multiple sets of TiDB Operator, the following parameters ar
     appendReleaseSuffix: true
     scheduler:
       create: false
+    advancedStatefulset:
+      create: false
+    admissionWebhook:
+      create: false
     ```
 
     > **Note:**
     >
     > * It is recommended to deploy the new TiDB Operator in a separate namespace.
     > * Set `appendReleaseSuffix` to `true`.
     > * If you configure `scheduler.create: true`, a `tidb-scheduler` named `{{ .scheduler.schedulerName }}-{{.Release.Name}}` is created. To use this `tidb-scheduler`, you need to configure `spec.schedulerName` in the `TidbCluster` CR to the name of this scheduler.
+    > * You need to set `advancedStatefulset.create: false` and `admissionWebhook.create: false` because you cannot deploy multiple sets of AdvancedStatefulSet controller and AdmissionWebhook.
```

Comment:
```suggestion
    > * You need to set `advancedStatefulset.create: false` and `admissionWebhook.create: false`, because deploying multiple sets of AdvancedStatefulSet controller and `tidb-admission-webhook` is not supported.
```